src/components/modules/paste.ts

```diff
@@ -434,9 +435,20 @@ export default class Paste extends Module {
       return;
     }
 
-    event.preventDefault();
-    this.processDataTransfer(event.clipboardData);
+    /**
+     * If no block selected, skip processing of paste event.
+     */
+    if (!BlockSelection.anyBlockSelected) {
+      return;
+    } else {
+      /**
+       * process multi-block selection paste.
+       */
+      dataTransfer = await this.processMultiBlockSelection(dataTransfer);
```

Comment:
`anyBlockSelected` does not guarantee that there are several blocks. There could be the single block selected.